keepassxc-browser/background/page.js

```diff
@@ -1,6 +1,8 @@
 'use strict';
 
 const defaultSettings = {
+    afterFillSorting: SORT_BY_DEFAULT,
+    afterFillSortingTotp: SORT_BY_DEFAULT,
```

Comment:
Shouldn't TOTP sort by relevant by default?

Reply:
Sure. Let's do that.